apps/ocas/src/act_allow.erl

```diff
@@ -0,0 +1,65 @@
+-module(act_allow).
+%%%-------------------------------------------------------------------
+%%% @author Duncan Sparrell
+%%% @copyright (C) 2015, sFractal Consulting LLC
+%%%
+%%% All rights reserved.
+%%%
+%%% Redistribution and use in source and binary forms, with or without
+%%% modification, are permitted provided that the following conditions are
+%%% met:
+%%%
+%%% * Redistributions of source code must retain the above copyright
+%%%   notice, this list of conditions and the following disclaimer.
+%%%
+%%% * Redistributions in binary form must reproduce the above copyright
+%%%   notice, this list of conditions and the following disclaimer in the
+%%%   documentation and/or other materials provided with the distribution.
+%%%
+%%% * The names of its contributors may not be used to endorse or promote
+%%%   products derived from this software without specific prior written
+%%%   permission.
+%%%
+%%% THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+%%% "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+%%% LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
+%%% A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
+%%% OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+%%% SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+%%% LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES. LOSS OF USE,
+%%% DATA, OR PROFITS. OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+%%% THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+%%% (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+%%% OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+%%%-------------------------------------------------------------------
+
+-author("Duncan Sparrell").
+-license("Apache 2.0").
+
+-export([ allow_server/1 ]).
+
+%% This routine API handles all the actions that can be taken
+
+allow_server(State) ->
+    %% separate process to handle allow action
+
+    %% initialize
+    lager:debug( "starting allow_server with ~p", [State] ),
+
+    %% await messages, then process them
+    receive
+        %% keepalive (for testing)
+        { From, keepalive } ->
+            lager:debug( "allow_server got keepalive" ),
+            From!{keepalive_received, allow_server},
+            allow_server(State);
+        %% stop server - note it doesnt loop
+        stop_server ->
+            lager:debug( "allow_server stopping" ),
+            stopping;
+        %% handle unaccounted for message
+        _ ->
+            lager:debug( "allow_server server got something not accounted for" ),
```

Comment:
Line 62 is too long:             lager:debug( "allow_server server got something not accounted for" ),.